### PR TITLE
New package: COVERCorp.Holoearth version 1.1.0

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132195)

### CODE REVIEW

**Comment by solomoncyj** [comment]
it worked for me

**Comment by Trenly** [comment]
> it worked for me

You'll probably have to specify the exact silent switches in the `InstallerSwitches`

**Comment by solomoncyj** [comment]
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20240101121539
validation png does not help

File: manifests/c/COVERCorp/Holoearth/1.1.0/COVERCorp.Holoearth.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: COVERCorp.Holoearth
+PackageVersion: 1.1.0
+InstallerType: inno
+InstallerSwitches:         	 	
+      Silent: /SP- /SUPPRESSMSGBOXES /VERYSILENT
+      SilentWithProgress:  /SP- /SUPPRESSMSGBOXES /SILENT
+Installers:
+- InstallerUrl: https://asset-prd.lobby.holoearth.com/launcher/v1.1.0/d5e2a8a9-b166-4280-b124-b0110431eec4/HoloearthSetup.exe
+  Architecture: x86
```
Comment (Trenly) [inline manifests/c/COVERCorp/Holoearth/1.1.0/COVERCorp.Holoearth.installer.yaml:12]:
```suggestion
  Architecture: x64
```
```
2024-01-01 04:15:39.339   64-bit install mode: No
2024-01-01 04:15:39.339   Defaulting to OK for suppressed message box (OK):
                          このプログラムは
                          
                          x64プロセッサー向けの Windows にしかインストールできません。
2024-01-01 04:15:39.339   Got EAbort exception.
2024-01-01 04:15:39.339   Deinitializing Setup.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by solomoncyj** [comment]
Resolved

**Comment by stephengillie** [comment]
> InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20240101121539 validation png does not help

Unfortunately, the orchestration console frequently covers the installer. One of the benefits of Manual Validation is having an interactive session, to bring the installer to the top before taking a screenshot. But Manual Validation doesn't scale enough.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**